src/main/java/uk/gov/hmcts/ccd/endpoint/std/GlobalSearchEndpoint.java

```diff
@@ -109,7 +119,18 @@ public GlobalSearchResponsePayload searchForCases(@RequestBody @Valid GlobalSear
         log.debug("GlobalSearchEndpoint.searchForCases execution completed in {} milliseconds...", between.toMillis());
 
         return result;
+    }
+
+    public static String buildCaseIds(GlobalSearchResponsePayload globalSearchResult) {
+        return globalSearchResult.getResults().stream().limit(MAX_CASE_IDS_LIST)
+            .map(c -> String.valueOf(c.getCaseReference()))
+            .collect(Collectors.joining(CASE_ID_SEPARATOR));
+    }
 
+    public static String buildCaseTypeIds(GlobalSearchResponsePayload globalSearchResult) {
```

Comment:
Both of these methods are identical other than the function called in the map ?

Could have a single method and pass it two separate functions - one for case reference and one for case type ud

Reply:
fixed in fixed in [cfb1298](https://github.com/hmcts/ccd-data-store-api/pull/1672/commits/cfb129896c8f8d8d4c56b4f0af2222221d6c5474) one has been changed as i did not need one of the functions